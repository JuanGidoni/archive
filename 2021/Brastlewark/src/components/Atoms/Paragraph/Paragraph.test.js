import React from 'react';
import renderer from 'react-test-renderer';
import Paragraph from './Paragraph';

describe('<Paragraph />', () => {
 it('renders correctly and match snapshot', () => {
  const tree = renderer
   .create(<Paragraph text="Example" />)
   .toJSON();
  expect(tree).toMatchSnapshot();
 });
});
