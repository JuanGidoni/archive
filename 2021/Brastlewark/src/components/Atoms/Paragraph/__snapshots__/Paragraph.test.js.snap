// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> renders correctly and match snapshot 1`] = `
<p>
  Example
</p>
`;
